ios/RCTWebRTC/WebRTCModule+RTCDataChannel.m

```diff
@@ -33,6 +49,7 @@ @implementation WebRTCModule (RTCDataChannel)
       || dataChannel.readyState == RTCDataChannelStateOpen)) {
     dataChannel.peerConnectionId = peerConnectionId;
     NSNumber *dataChannelId = [NSNumber numberWithInteger:config.channelId];
+    dataChannel.originDataChannelId = dataChannelId;
```

Comment:
What happens if no channelId was passed when creating the data channel? Won't this be null?

Reply:
See [here](https://github.com/react-native-webrtc/react-native-webrtc/blob/master/RTCPeerConnection.js#L377), we'll always make sure passing a config with id.
At the same time, `config.channelId` is in type `int`, which always has value instead of a Class type.

Reply:
> What happens if no channelId was passed when creating the data channel? Won't this be null?

See [here](https://github.com/react-native-webrtc/react-native-webrtc/blob/master/RTCPeerConnection.js#L377), we'll always make sure passing a config with id.
At the same time, `config.channelId` is in type `int`, which always has value instead of a Class type.

Reply:
@saghul Any comment on this?